pkgs/development/libraries/libgda/default.nix

```diff
@@ -24,8 +25,8 @@ assert postgresSupport -> postgresql != null;
   hardeningDisable = [ "format" ];
 
   nativeBuildInputs = [ pkgconfig intltool itstool libxml2 gobject-introspection vala ];
-  buildInputs = with stdenv.lib; [ gtk3 openssl libgee ]
-    ++ optional (mysqlSupport) libmysqlclient
+  buildInputs = with stdenv.lib; [ gtk3 openssl libgee libmysqlclient]
```

Comment:
After reading this comment: https://github.com/NixOS/nixpkgs/issues/63110#issuecomment-502002955

I tried moving libmysqlclient to the list of inputs and it worked.